cli.js

```diff
@@ -17,4 +17,15 @@ var cli = meow({
 	string: ['_']
 });
 
-cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite});
+function errorHandler(err) {
+	if (err) {
+		console.error(err.message);
+		process.exit(1);
+	}
+}
+
+try {
+	cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite}, errorHandler);
+} catch (err) {
+	errorHandler(err);
```

Comment:
I've used it twice: as `cb` for the `cpy` call and within the `catch`, and I wanted to be [DRY](http://en.wikipedia.org/wiki/Don%27t_repeat_yourself) ;-)